operating/default-security-policies.adoc

```diff
@@ -9,6 +9,15 @@ toc::[]
 [role="_abstract"]
 The default security policies in {product-title} provide broad coverage to identify security issues and ensure best practices for security in your environment. By configuring those policies, you can automatically prevent high-risk service deployments in your environment and respond to runtime security incidents.
 
+[NOTE]
+====
+The severity levels for policies in {product-title} are different from the severity levels that are assigned by Red Hat Product Security.
+
+The policy severity levels are Critical, High, Medium, and Low. Red Hat Product Security rates severity levels as Critical, Important, Moderate, and Low.
```

Comment:
May want to put "RHACS" before "policy severity levels" just to avoid confusion.